src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/PathConverter.cs

```diff
@@ -20,14 +28,17 @@ internal class PathConverter : IPathConverter
     // are inverted, it sends us their local path, and thinks about our local path as remote.
     private readonly string _originalPath = "";
 
-    public PathConverter(string originalPath, string deploymentPath, IFileHelper fileHelper)
+    public PathConverter(string originalPath!!, string deploymentPath!!, IFileHelper fileHelper!!)
     {
         _originalPath = fileHelper.GetFullPath(originalPath).TrimEnd('\\').TrimEnd('/') + Path.DirectorySeparatorChar;
         _deploymentPath = fileHelper.GetFullPath(deploymentPath).TrimEnd('\\').TrimEnd('/') + Path.DirectorySeparatorChar;
     }
 
-    public string UpdatePath(string path, PathConversionDirection updateDirection)
+    public string? UpdatePath(string? path, PathConversionDirection updateDirection)
     {
+        if (path == null)
```

Comment:
it actually happens. I did not look too much into it because I want to get this into a working state first where we can deploy and see if that works in reality. It will be used for just a single scenario for now.